### PR TITLE
Added code to clear the text

### PR DESCRIPTION
**Pull request by AnthonyHarwood**
I noticed the text in the error view was not being cleared after the error was corrected.  For example, "Username is too short". 

I can think of a few different ways this could have been fixed.  I try to keep it simple.  If you prefer a different approach to resolving this, please don't hesitate to let me know.

Thanks!

### CODE REVIEW

File: core/text_view.js
Comment (cognitivegears) [inline core/text_view.js:1]: This looks like a great change! See below for one small comment -

File: core/text_view.js [hunk 1]
```diff
@@ -179,6 +179,10 @@ TextView.prototype.setText = function (text, redraw) {
 };
 
 TextView.prototype.clearText = function () {
+    if (this.text) {
+        this.setText(' '.repeat(this.text.length));
```
Comment (cognitivegears) [inline core/text_view.js:183]: Can we use `this.fillChar` here instead of a literal space?

**Review by AnthonyHarwood — COMMENTED** [review]
Good suggestion.  Thanks.

**Review by cognitivegears — APPROVED** [review]
Looks good to me, I'll wait for @NuSkooler in case he has any concerns but thanks so much for the PR, keep 'em coming :)

**Review by NuSkooler — APPROVED** [review]
Awesome, I like simple! Thanks!

**Pull request merged**